gflownet/evaluator/base.py

```diff
@@ -275,6 +279,14 @@ def compute_log_prob_metrics(self, x_tt, metrics=None):
 
         if "reward_batch" in reqs:
             rewards_x_tt = self.gfn.proxy.rewards(self.gfn.env.states2proxy(x_tt))
+            log_rewards_x_tt = torch.log(
+                tfloat(
+                    rewards_x_tt,
+                    float_type=self.gfn.float,
+                    device=self.gfn.device,
+                )
+            )
```

Comment:
Did you use `tfloat()` here for any particular reason? It seems to be that `torch.log(rewards_x_tt)` is enough.